dev/nm/datarepo-api.yaml

```diff
@@ -19,6 +17,10 @@ env:
   TERRA_COMMON_TRACING_STACKDRIVER_EXPORT_ENABLED: false
   OPENCENCUS_SPRING_ENABLED: false
   GOOGLE_SECURE_FOLDER_RESOURCE_ID: "753276429356"
+  OIDC_CLIENTID: 970791974390-1581mjhtp2b3jmg4avhor1vabs13b7ur.apps.googleusercontent.com
+  OIDC_AUTHORITYENDPOINT: https://accounts.google.com
+  OIDC_ADDCLIENTIDTOSCOPE: false
+  OIDC_EXTRAAUTHPARAMS:
```

Comment:
Is this a legal assignment?

Reply:
Weirdly, yeah.  I had to double check when I put it in.  We do assignments like this in other places too.  I agree...it's a little jarring

Reply:
Eh, decided to set it to an empty string explicitly...it feels less gross that way